tests/component/encapsulation.cpp

```diff
@@ -0,0 +1,31 @@
+#include "gtest/gtest.h"
+
+#include <libcellml>
+
+
+TEST(Encapsulation, serialise) {
+    libcellml::Component parent = libcellml::Component();
```

Comment:
I hope <code>libcellml::Component parent;</code> is pretty clear to all C++ programmers about what is happening.